### PR TITLE
CICD-8002: Update to new python version 3.9+

### PR DESCRIPTION
**Pull request by abhijeetsonawane90**

### CODE REVIEW

**Review by gideonw — APPROVED** [review]
Lgtm, I would wait for one more review

File: setup.py
Comment (andyg-zg) [inline setup.py:63]: We need to support all active versions of Python so this should be 3.9 and greater.

Comment (abhijeetsonawane90) [reply setup.py]: Yes, I chaged the python version to >=3.9. Also the changes are tested with Python version 3.9, 3.10, 3.11, 3.12 which are supported.

**Pull request merged**
